apps/landing-page/package.json

```diff
@@ -11,24 +11,22 @@
     "lint": "eslint --fix ."
   },
   "dependencies": {
-    "@mantine/core": "^7.6.2",
-    "@mantine/form": "^7.6.2",
-    "@mantine/hooks": "^7.6.2",
+    "@mantine/core": "^7.12.0",
```

Comment:
end-all-progresd

Reply:
@cryptokin92 , thank you for your comment, could you please ellaborate what do you mean by a bit of explanation? Thank you!

With kind regards,
Team ParaSpell✨